torch/nn/functional.py

```diff
@@ -550,6 +550,19 @@ def bilinear(input1, input2, weight, bias=None):
         return Bilinear.apply(input1, input2, weight, bias)
 
 
+def embedding(input, weight, padding_idx=None,
+              max_norm=None, norm_type=2, scale_grad_by_freq=False,
+              sparse=False):
+    assert not sparse, 'sparse not supported in nn.functional.embedding for now'
```

Comment:
You can check how the [`nn.Module` version](https://github.com/pytorch/pytorch/blob/73fead9f8f4a53e64a078d3891ae43e865b32c7b/torch/nn/modules/sparse.py#L87-L94) works but I think this is completely handled by the underlying `Function`.